app/konnect/gateway-manager/plugins/update-custom-plugin.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Editing or deleting a custom plugin's schema
+content_type: how-to
+---
+
+The workflow for updating an in-use custom plugin depends on whether you need to update the schema:
+
+* **No change to plugin schema:** Editing a custom plugin's logic **without** any change in 
+schema doesn't cause the control plane to go out of sync with the data planes. 
+
+  In this situation, you only need to make sure that each data plane node has the correct logic. 
+  The schema on the control plane doesn't need to be updated.
+
+* **Changes to plugin schema:** If the schema needs to change, you must update both the schema in 
+{{site.konnect_short_name}} and the plugin code itself on each data plane node.
+
+There is no versioning for custom plugins. If you need to version a schema 
+(that is, maintain two or more similar copies of a custom plugin), upload it as a new custom plugin.
```

Comment:
We might need to clarify that all custom plugin schemas need a unique name in Konnect...